.github/workflows/build_and_deploy.yaml

```diff
@@ -1,5 +1,7 @@
 name: Build and Deploy
 on:
+    push:
+        branches: ["t3chguy/fix/18486"]
```

Comment:
This would need to be changed before merge, right?

Reply:
Well it'd cause no harm, but yes it was in for testing, will remove, thanks for spotting!